src/components/attendee/AttendeeLayout.tsx

```diff
@@ -12,7 +12,7 @@ import { LoadingPage } from "components/molecules/LoadingPage";
 
 import { NotLoggedInFallback } from "components/atoms/NotLoggedInFallback";
 
-import { AttendeeHeader } from "./Header/AttendeeHeader";
+import { AttendeeHeader } from "./AttendeeHeader/AttendeeHeader";
```

Comment:
These components would be good to have their own `index.ts`, be moved into something like `./src/components/attendee/` or `./src/components/attendee/layout` and be imported with absolute, not relative paths.

Reply:
[2ebc1f3](https://github.com/sparkletown/sparkle/pull/2827/commits/2ebc1f31eb8b1121e844036f240cd927eeb6d714)